lib/openstudio-standards/standards/ashrae_90_1_prm/ashrae_90_1_prm_2019/ashrae_90_1_prm_2019.rb

```diff
@@ -18,7 +18,7 @@ def load_standards_database(data_directories = [])
   #
   # @return [Double] the baseline infiltration rate, in cfm/ft^2
   # defaults to no infiltration.
-  def space_infiltration_rate_75_pa
+  def prm_space_infiltration_rate_75_pa
```

Comment:
I'm not sure if this name change is useful since the function is already in a PRM specific class.

Reply:
The initial function name is overlapping with the function in [space_infiltration_rate_75_pa](https://github.com/NREL/openstudio-standards/blob/d0e3fabaeda812213835d134cd03feacec04ce4d/lib/openstudio-standards/standards/Standards.Space.rb#L1345) in the Standards.Space.rb.

The confusion becomes the `space_infiltration_rate_75_pa` in Standards.Space.rb has one argument but this function does not. Ruby also does not supporting function overloading so I worried that this might not right Ruby way.

Rubymine did called out the missing argument error though no running errors when complied to run.

Let me know if the original function name is intentional.

Reply:
I think that we could add the argument to be consistent with the methods in Standards. The argument doesn't seem to actually be used, it just seemed to be added because the methods is in `***.Space.rb`.

Reply:
That is OK with me too.
One thing I am not clear is though the function in `***.Space.rb` has a `Space` argument. But none of the implementations, including overriding functions use this argument.

My suggestion will be either remove this space argument in the parent class, or set the space to a default nil. I set the space to nil in the second implementation.

Reply:
@mdahlhausen - Is there a reason for that (`space` argument for the `space_infiltration_rate_75_pa` method is provided but not used)?

Reply:
I think it's just to align with the naming convention `object_some_method(object)`.

We don't need it; I'm fine to get rid of the space argument.  Or you could just pass in nil wherever you call it in the PRM workflow.

Also, in response to the above comments, ruby does overload functions - that how most of the standards code works - loading initial methods, then overloading them for each standard.

Reply:
@mdahlhausen Thanks, I put nil in case in the future Space is needed.

Most cases I see in the code base is overriding the function. Could you show me an example of such overloading in the code base? From what I read, Ruby does not support [function overloading](https://www.geeksforgeeks.org/method-overloading-in-ruby/).

Reply:
Overloading - you are right in that ruby doesn't support argument overloading, but you can overload the method with the same arguments.

Glad the nill solution will work fine.